src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/NamingStyleOptionPageControl.xaml

```diff
@@ -43,13 +43,16 @@
             <KeyBinding Key="Down" Modifiers="Alt" Command="{StaticResource MoveDown}" />
         </Grid.InputBindings>
         <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="5" />
```

Comment:
❓ Why two new rows here?

Reply:
The two TextBlocks didn't look right so close to each other so added a 5 unit space between.  I can add the 5 units it to the top margin of the 2nd on if you prefer.

Reply:
I moved the spacing to the Margin setting.